Cannot find module 'mkdirp'

Hi.

Maybe it's a very basic thing and i'm just missing something, but can't make it run.
I have very simple test case and it runs perfectly when i run: `testcafe chrome e2e/test.js`

But when i try to run this grunt task:
```javascript
testcafe: {
      test: {
        options: {
          files: ['e2e/*.js'],
          browsers: ['chrome'],
          takeScreenshotsOnFail: true,
          screenshotsPath: 'e2e/screenshots',
        }
      }
    }
```

I get error:
`Loading "testcafe.js" tasks...ERROR
Error: Cannot find module 'mkdirp'`

What am i missing here? Should i install `mkdirp` as dev dependency?

node 4.7.2
testcafe 0.15.0 (tried both - global and local)
grunt 0.4.5
grunt-testcafe 0.15.0

--- a/package.json
+++ b/package.json
@@ -33,12 +33,14 @@
     "grunt-contrib-clean": "1.1.0",
     "grunt-http-server": "2.0.0",
     "gruntify-eslint": "3.1.0",
-    "mkdirp": "0.5.1",
     "testcafe": "0.15.0",
     "testcafe-reporter-xunit": "2.1.0"
   },
   "peerDependencies": {
     "grunt": "0.4.0 - 1",
     "testcafe": "~0.15.0"
+  },
+  "dependencies": {
+    "mkdirp": "0.5.1"
   }
 }

